### ISSUE TITLE
Allow Android PWAs to swap between desktop/mobile view

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#37265]
## Description 
As it stands right now, PWAs ("Installed apps") on Android open and display in the format that reflects the view set in the main browser. While this makes sense as to why it works this way, I think users would appreciate the ability to swap between different views on the PWA itself instead of having to open the main browser app to swap (which defeats the purpose of the PWA).

![image](https://github.com/brave/brave-browser/assets/3299098/93a8efb3-f95b-41a8-b85f-e0d7336c8b31)

### Additional information

One such user request:
https://community.brave.com/t/brave-pwa-apps-to-load-as-desktop-mode/538665/12

**Comment by Krylanc3lo** [comment]
@Descatres 
I noticed the same issue, I cannot install new PWA on Brave but still able on vivaldi. Were you able to do it finally ?

Thank you!

**Comment by Descatres** [comment]
> @Descatres I noticed the same issue, I cannot install new PWA on Brave but still able on vivaldi. Were you able to do it finally ?
> 
> Thank you!

Not really... I can only use the apps I previously installed. I've even tried with multiple versions of android (12 and 14) to test if the apis would make a difference (just for the sake of trying) but they did not - I was not able to install on neither of them.
Maybe it is possible to create a repo with some of them but Idk tbh.
Are you still able to do it on Vivalvi?
